src/arch/arm/32/head.S

```diff
@@ -153,9 +153,12 @@ BEGIN_FUNC(_start)
 
     /* Hyp kernel always run in Hyp mode. */
 #ifndef CONFIG_ARM_HYPERVISOR_SUPPORT
-    /* Initialise ABORT stack pointer */
     cps #PMODE_ABORT
-    ldr sp, =_breakpoint_stack_top
+    /* Initialise ABORT stack pointer.
+     * For ARM, the `sp' register is banked between PMODEs, so we configure
+     * a stack for when we take a exception (see traps.S). We then load the
+     * real kernel stack. Cross reference to `arm_data_abort_exception'. */
+    ldr sp, =_abort_stack_top
```

Comment:
I'd prefer the comment to go above the `csp #PMODE_ABORT`. Should be "an exception".